editor/src/components/editor/store/store-deep-equality-instances.ts

```diff
@@ -3133,6 +3129,13 @@ export const ModalDialogKeepDeepEquality: KeepDeepEqualityCall<ModalDialog> = (
       } else {
         return keepDeepEqualityResult(oldValue, false)
       }
+    case 'disconnect-github-project':
+      if (newValue.type === oldValue.type) {
+        return keepDeepEqualityResult(newValue, true)
+      } else {
+        return keepDeepEqualityResult(oldValue, false)
+      }
+      break
```

Comment:
Interesting. This appears to be the wrong way around (if the values are equal it should be returning `keepDeepEqualityResult(oldValue, true)`, but I see that you've copied the above (presumably also incorrect) case statement

Reply:
https://github.com/concrete-utopia/utopia/pull/2794/commits/89e7b97d478bf0a16f77f6003adbfca6d31821d9